### DISCUSSION TITLE
What is your major in your former university?

### DISCUSSION THREAD

**Discussion by undeux1016** [NAIST-SE/AttractingContributors#936] [2023 Attraction Contributions]
Students in NAIST come from a variety of backgrounds. While many of them majored in information science, I am curious about the majors of others. Would you please tell me your major in your former university?

https://github.com/undeux1016/AttractingContributors/tree/undeux1016-patch-1

**Comment by bconfessor** [comment]
Done! Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
